tests/bears/DebugBearsTest.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import sys
+
+from io import StringIO
+
+from coalib.bears.Bear import db, debug_run
+
+
+class DebugBearsTest(unittest.TestCase):
+
+    def test_debug_run(self):
+        args = ('a', ('b', 'c'))
+        kwargs = {'d': 'e'}
+
+        def fun(*args, **kwargs):
+            yield 1
+            yield 2
+            yield 3
+
+        input = StringIO('q\nc\nq\nc')
+        output = StringIO()
+        old_trace = sys.gettrace()
+        dbg = db(stdin=input, stdout=output)
+        dbg.do_q = dbg.do_continue
+        self.assertEqual(debug_run(fun, dbg, *args, **kwargs), [1, 2, 3])
+        output = output.getvalue()
+        sys.settrace(old_trace)
+        # restore the coverage settrace to prevent the coverage breakage
+        # on project because  we can't chain coverage trace to run parallel
+        # with debugger. To increase the coverage Mock test has been added in
+        # BearTest file.
+        self.assertEqual(output.split('\n')[1], '-> yield 1')
+        self.assertEqual(output.split('\n')[3], '-> yield 2')
+        self.assertEqual(output.split('\n')[5], '-> yield 3')
```

Comment:
more tests please :)
- `run()` that doesn't yield, but returns an `Iterable`
- `run()` that returns a generator.